src/picknik_dual_ur_arm_config/waypoints/ur_waypoints.yaml

```diff
@@ -0,0 +1,79 @@
+- description: 'Both arms are in a down position'
+  favorite: true
+  joint_state:
+    effort: []
+    header:
+      frame_id: world
+      stamp:
+        nanosec: 0
+        sec: 0
+    name:
+    - left_shoulder_pan_joint
+    - left_shoulder_lift_joint
+    - left_elbow_joint
+    - left_wrist_1_joint
+    - left_wrist_2_joint
+    - left_wrist_3_joint
+    - right_shoulder_pan_joint
+    - right_shoulder_lift_joint
+    - right_elbow_joint
+    - right_wrist_1_joint
+    - right_wrist_2_joint
+    - right_wrist_3_joint
+    position:
+    - 8.581502470187843e-05
+    - -1.5136764603659916
+    - -1.799275792510518
+    - -1.5993562600093492
+    - 1.6699146867751609
+    - -1.3363372068852196e-06
+    - -6.064204568974674e-05
+    - -1.5136764603659916
+    - -1.71359599286716
+    - -1.6564761264382546
+    - 1.6699516917224972
+    - -4.144793883897364e-05
+    velocity: []
+  joint_group_names: [dual_arm_manipulator, manipulator_left, manipulator_right]
+  name: Arms Down
+- description: ''
+  favorite: true
+  joint_state:
+    effort: []
+    header:
+      frame_id: world
+      stamp:
+        nanosec: 0
+        sec: 0
+    name:
+    - left_shoulder_pan_joint
+    - left_shoulder_lift_joint
+    - left_elbow_joint
+    - left_wrist_1_joint
+    - left_wrist_2_joint
+    - left_wrist_3_joint
+    - right_shoulder_pan_joint
+    - right_shoulder_lift_joint
+    - right_elbow_joint
+    - right_wrist_1_joint
+    - right_wrist_2_joint
+    - right_wrist_3_joint
+    position:
+    - 0
+    - -1.41
+    - -0.7
+    - -2.12
+    - 1.67
+    - 0
+    - 0
+    - -1.41
+    - -0.7
+    - -2.12
+    - 1.67
+    - 0
+    velocity: []
+  joint_group_names:
+  - dual_arm_manipulator
+  - manipulator_left
+  - manipulator_right
+  name: Home
```

Comment:
I also think the formatter didn't like the missing newline

Reply:
Created an issue: https://github.com/PickNikRobotics/moveit_studio/issues/6782